web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -62,34 +60,80 @@ public static void setUp() {
             throw new IllegalStateException(TRANSIT_DATA_DIR + " directory should not already exist.");
         }
         Preconditions.checkState(transitDataDir.mkdir(), "could not create directory " + TRANSIT_DATA_DIR);
-    }
 
-    @AfterAll
-    public static void cleanUp() {
-        Helper.removeDir(new File(TARGET_DIR));
-        Helper.removeDir(new File(TRANSIT_DATA_DIR));
-    }
-
-    @Test
-    public void testExportEndToEnd() throws IOException {
         // Setup necessary mock
         final JarLocation location = mock(JarLocation.class);
         when(location.getVersion()).thenReturn(Optional.of("1.0.0"));
 
         // Add commands you want to test
-        final Bootstrap<GraphHopperServerConfiguration> bootstrap = new Bootstrap<>(new GraphHopperApplication());
+        bootstrap = new Bootstrap<>(new GraphHopperApplication());
         bootstrap.addBundle(new GraphHopperBundle());
         bootstrap.addCommand(new ImportCommand());
         bootstrap.addCommand(new ExportCommand());
 
         // Build what'll run the command and interpret arguments
-        Cli cli = new Cli(location, bootstrap, System.out, System.err);
+        cli = new Cli(location, bootstrap, System.out, System.err);
         cli.run("import", "test-data/beatty-sample-feed-config-car.yml");
+    }
+
+    @AfterAll
+    public static void cleanUp() {
+        Helper.removeDir(new File(TARGET_DIR));
+        Helper.removeDir(new File(TRANSIT_DATA_DIR));
+    }
+
+    @Test
+    public void testExportEndToEnd() throws IOException {
         cli.run("export", "test-data/beatty-sample-feed-config-car.yml");
         CSVFormat format = StreetEdgeExporter.CSV_FORMAT;
         File expectedOutputLocation = new File(TARGET_DIR + "street_edges.csv");
         CSVParser parser = CSVParser.parse(expectedOutputLocation, StandardCharsets.UTF_8, format);
         List<CSVRecord> records = parser.getRecords();
         assertEquals(769, records.size());
     }
+
+    @Test
+    public void testExportSingleRecord() {
+        GraphHopperServerConfiguration config = new GraphHopperServerConfiguration();
```

Comment:
@mark-idleman So, I was able to remove all this stuff and run the previous test, but I had more trouble when I wanted to use a config object directly in my code. If there's a quick-ish way to point to a yaml path and get a config back then I'd be happy to use it, but I couldn't find it easily by stepping through code.